kasa/cli.py

```diff
@@ -330,6 +366,11 @@ async def state(dev: SmartDevice):
     echo(f"[bold]== {dev.alias} - {dev.model} ==[/bold]")
     echo(f"\tHost: {dev.host}")
     echo(f"\tPort: {dev.port}")
+    if dev.credentials is not None:
+        if dev.credentials.username is not None:
+            echo(f"\tUsername: {dev.credentials.username}")
+        if dev.credentials.password is not None:
+            echo(f"\tPassword: {dev.credentials.password}")
```

Comment:
fixed

Reply:
There's no output anymore at all, but the cli test still tests for it.

Reply:
The test is testing that cli passes the credentials to the device.  Because the credentials aren't in the standard state output anymore the test patches the state function so this works ok.

Reply:
Oh, I see, I somehow missed that. Let's get this merged so we can move forward!